pkg/volume/awsebs/aws_util.go

```diff
@@ -125,6 +128,39 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner, node *
 	return name, volumeOptions.CapacityGB, labels, fstype, nil
 }
 
+// getCandidateZones finds possible zones that a volume can be created in
+func getCandidateZones(cloud *aws.Cloud, selectedNode *v1.Node, pvcName string) (sets.String, error) {
```

Comment:
Lets add some unit tests if we can...

Reply:
I couldn't find a good way to mock `aws.Cloud` outside its package, so only added test cases for delayed binding test case. It's probably cloud provider's responsibility to provide reasonable mocks so I left a TODO here. please check

Reply:
I think that is fine. `GetCandidateZonesForDynamicVolume` should already be considered "tested".